src/corelibs/U2Gui/src/util/SaveDocumentController.cpp

```diff
@@ -309,7 +309,9 @@ void SaveDocumentController::addFormatExtension(QString& path) const {
 
 QString SaveDocumentController::prepareDefaultFileFilter() const {
     bool useGzipVariant = conf.compressCheckbox && conf.compressCheckbox->isEnabled();
-    return FileFilters::createFileFilter(currentFormatName, formatsInfo.getExtensionsByName(currentFormatName), useGzipVariant);
+    return FileFilters::createSingleFileFilter(currentFormatName,
```

Comment:
createSingleFileFilter will not create `All files(*.*)` filter right? 

In this case this solution is wrong: we should have `All files (*.*)` filter in every file open/save dialog, because formats in UGENE are not extension based but content (header) based.

Limited per extension filters should be considered as helpers, but not restrictions.

Reply:
> createSingleFileFilter will not create `All files(*.*)` filter right?

Yes.

> In this case this solution is wrong: we should have All files (*.*) filter in every file open/save dialog, because formats in UGENE are not extension based but content (header) based.

I didn't touch it. I corrected the default filter setting. It should be one.

Reply:
Right. Resolving